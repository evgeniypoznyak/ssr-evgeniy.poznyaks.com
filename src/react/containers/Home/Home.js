import React from 'react';
import {makeStyles} from '@material-ui/core/styles';
import Avatar from '@material-ui/core/Avatar';
import Grid from '@material-ui/core/Grid';
import {Bio} from '../../shared/initialData/bio';

const useStyles = makeStyles(theme => ({

    avatar: {
        margin: 10,
    },
    bigAvatar: {
        margin: 20,
        width: 100,
        height: 100,
    },
}));

const Home = props => {
    const classes = useStyles();
    return (
        <div>
            <Grid container justify="flex-end">
                <Avatar alt="Evgeniy Poznyak" src={'/assets/me.jpg'} className={classes.bigAvatar}/>
            </Grid>
            {Bio}
        </div>
    );
};

export default Home;
