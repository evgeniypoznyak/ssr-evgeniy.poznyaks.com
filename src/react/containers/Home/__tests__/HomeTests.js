import React from 'react';
import Home from '../Home';
import renderer from 'react-test-renderer';
// import {shallow} from 'enzyme';
// setup file
import {configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
configure({adapter: new Adapter()});

// test file
import { shallow, mount, render } from 'enzyme';

describe('Home', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<Home/>).toJSON();
        expect(tree).toMatchSnapshot();
        expect(tree.children.length).toBe(2);
    });

    it('should render correctly', () => {
        const wrapper = shallow(<Home/>);
        console.log(wrapper);
    });
});
