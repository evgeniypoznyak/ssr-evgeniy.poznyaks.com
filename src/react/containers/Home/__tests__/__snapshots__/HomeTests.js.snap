// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div>
  <div>
    Welcome
  </div>
  <div>
    Welcome Again
  </div>
</div>
`;
